src/components/Person/Person.jsx

```diff
@@ -1 +1,36 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import Proptypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partner = sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+
+      {age
+        && (<p className="Person__age">{`I am ${age}`}</p>)}
+
+      <p className="Person__partner">
+        {isMarried
```

Comment:
More correctly move this condition from jsx to variable for prevent jsx code clogging